aleph2_data_model/src/com/ikanow/aleph2/data_model/objects/data_import/DataSchemaBean.java

```diff
@@ -783,6 +788,45 @@ public String service_name() {
 		}		
 	}
 	
+	public static class LoggingSchemaBean implements Serializable {
```

Comment:
Ah minor change, I was assuming LoggingSchemaBean would go in ManagementSchemaBean as a sub-bean

ie ManagementSchemaBean / LoggingSchemaBean <-> DataSchemaBean / SearchIndexSchemaBean